app/actions/remote/user.ts

```diff
@@ -451,3 +452,22 @@ export const unsetCustomStatus = async (serverUrl: string) => {
 
     return {data: true};
 };
+
+export const setDefaultProfileImage = async (serverUrl: string, userId: string) => {
+    let client: Client;
+
+    try {
+        client = NetworkManager.getClient(serverUrl);
+    } catch (error) {
+        return {error};
+    }
+
+    try {
+        await client.setDefaultProfileImage(userId);
+        updateLocalUser(serverUrl, userId, {lastPictureUpdate: 0});
```

Comment:
this should not be 0, you want to set the Date.now() here

Reply:
Corrected